### DISCUSSION TITLE
Kubeplus IDP Integration

### DISCUSSION THREAD

**Discussion by Prageeth-Wickramaarachchi** [cloud-ark/kubeplus#1363] [Q&A]
Is there a way to integrate a identity provider like Cognito to kubeplus?

**Comment by devdattakulkarni** [comment]
@Prageeth-Wickramaarachchi Can you elaborate more on this requirement? Are you wanting to allow Cognito credentials to be used on the applications that are deployed by KubePlus? 

You can configure this in the application and expose the required parameters in the application's Helm chart. Any application-level accesses happen directly against the application endpoint that you would have configured (via Ingress or NodePort). KubePlus does not intercept such requests.
